src/Main.java

```diff
@@ -0,0 +1,38 @@
+import java.util.ArrayList;
+
+/**
+ * Created by avery on 12/6/16.
+ */
+public class Main {
+
+    public static void main(String[] args) {
+        ArrayList<Character> characters = new ArrayList();
```

Comment:
on the left-hand side of this assignment you should use List instead of ArrayList logically